app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
     <string name="app_name">My Application</string>
-    <string name="web_client_id">957415054330-j5fa1cl19qlthoshv7l2cccqv5v4icj4.apps.googleusercontent.com</string>
+    <string name="web_client_id">1043895872028–8m8f4kpcmqe60ofkef734vbgqph8g5ut.apps.googleusercontent.com</string>
```

Comment:
This should move to the other PR as well

Reply:
👍